### DISCUSSION TITLE
After upgrade to v4 getting JWEDecryptionFailed: decryption operation failed

### DISCUSSION THREAD

**Discussion by meliborn** [nextauthjs/next-auth#3476] [Help]
I have the next jwt config:
```	
jwt: {
  secret: process.env.JWT_SECRET,
  encryptionKey: process.env.JWE_KEY,
  encryption: true,
},
```
and after upgrade to v4 it stopped to work. What should I change to make it work again?

**Comment by balazsorban44** [comment]
https://next-auth.js.org/getting-started/upgrade-v4

**Reply by meliborn** [reply]
@balazsorban44 I saw that encryptionKey was removed from jwt object, but I can't figure out where to put it in the version 4

**Reply by balazsorban44** [reply]
Nowhere. That option has been removed. If you need to override the encoding/decoding algorithm (which in most cases you don't), just use the `encode` and `decode` options.

**Reply by meliborn** [reply]
@balazsorban44 but what to do with existing sessions?

**Reply by balazsorban44** [reply]
This has been a breaking change, so, technically not much to do if you use the default. Using the mentioned methods though, you should be able to decode old sessions as well, I think.
